### ISSUE TITLE
It should be possible but not required to include `rel=legend` links in the `<map-extent>` so that they can be parameterized by style etc

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#1053]
Currently this is not expected, and doesn't work, but it should be optional to do this.  Maybe if the link is provided in a `<map-extent>` the rendered label of the sub-layer should be the link instead of the layer label / title. tbd.
